website/templates/copyright.mako

```diff
@@ -1,7 +1,9 @@
+<% from datetime import datetime %>
+
 <div class="container copyright">
     <div class="row">
         <div class="col-md-12">
-            <p>Copyright &copy; 2011-2017 <a href="https://cos.io/">Center for Open Science</a> |
+            <p>Copyright &copy; 2011-${datetime.now().year} <a href="https://cos.io/">Center for Open Science</a> |
```

Comment:
We should use `utcnow` so that we're not relying on the server's timezone. I can change this when I merge.

Reply:
Ah okay. Thanks, @sloria.